docs/Project.toml

```diff
@@ -23,5 +23,5 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 BenchmarkTools = "1.3"
 CondaPkg = "0.2"
 Documenter = "0.27"
```

Comment:
It would be nice to bumper Documenter too.

Reply:
Oh I had thought I had done that already, well spotted, thanks!